apps/settings/js/panels/call/call_utils.js

```diff
@@ -0,0 +1,70 @@
+define(function() {
+  'use strict';
+
+  var mozSettings = window.navigator.mozSettings;
+  var conns = window.navigator.mozMobileConnections;
+
+  var CallUtils = {
+    /**
+     * Helper function. Stores settings into the database.
+     */
+    setToSettingsDB: function(settingKey, value, callback) {
+      var done = function done() {
+        if (callback) {
+          callback();
+        }
+      };
+
+      var getLock = mozSettings.createLock();
+      var request = getLock.get(settingKey);
+      request.onsuccess = function getFromDBSuccess() {
+        var currentValue = request.result[settingKey];
```

Comment:
This implementation might be out of dated and it seems not one is using the function now?